doaj/software-versions.yml

```diff
@@ -0,0 +1,125 @@
+---
+- name: Get software versions
+  hosts: monitor # replace with your target group or server(s)
```

Comment:
I'd prefer this to read `all` and then use `monitor` for only those with the `monitor` role, as per the `doaj-hosts.ini` file. You can also exclude by tag. e.g. `all:!test` being all non-test machines.